Added a section that explains Ember editions (Times 130) (#483)
ember-learn/ember-blog: source/2020-01-10-the-ember-times-issue-130.md

---

## [What is the Octane Edition? 🙋‍♀️🙋‍♂️](https://blog.emberjs.com/2019/12/20/octane-is-here.html)

Ember Octane introduces the idea of editions. Let's look at what an edition means in Ember and what features you can find in the Octane edition.

### What is an Edition?

In Ember, an [edition](https://emberjs.com/editions/) represents a **cohesive programming model** and can focus on a particular theme. For example, the Octane edition is focused on productivity and performance!

Having editions helps you and the Ember teams in many ways:

1. You can see the big picture of how features work together. Before editions were introduced, you might have had to stay on top of RFCs and release notes to understand why a feature was added or removed.

> 2. For people who don't follow Ember closely—that may be you, or people in the wider JavaScript world—it was easy to miss the significant improvements that happen over time. Editions help us market *tour de force* in Ember.

Author: ```suggestion
2. For people who don't follow Ember closely—that may be you, or people in the wider JavaScript world—it was <!--alex ignore easy-->easy to miss the significant improvements that happen over time. Editions help us market *tour de force* in Ember.
```

> 2. For people who don't follow Ember closely—that may be you, or people in the wider JavaScript world—it was "easy to miss the significant improvements that happen over time." Editions help us market *tour de force* in Ember.

Author: ```suggestion
2. For people who don't follow Ember closely—that may be you, or people in the wider JavaScript world—it was easy to miss the significant improvements that happen over time. Editions help us market *tour de force* in Ember.
```

3. Editions mark an opportunity to update Ember documentation, idioms, addons, and tooling (Ember Inspector, blueprints, codemods, and error messages). The end result is a cohesive, polished experience for you!

### What Features are in Octane Edition?

> Many new features cohere to form a new component model and a new reactivity system for Ember. These two big changes are keystone to Octane's ergonomic improvements.

Author: ```suggestion
Many new features cohere to form **a new component model and a new reactivity system** for Ember. These two big changes are keystone to Octane's ergonomic improvements.
```

You can find the full list of features in the [Ember Guides](https://guides.emberjs.com/release/upgrading/current-edition/#toc_what-is-ember-octane). We'll highlight a few major ones below.

> 1. **Glimmer components** offer a greatly simplified API, for both template and backing class. Say hi to native JavaScript classes!

Reviewer: As far as I know, Glimmer components may be built with classic component classes and not only native JavaScript classes. Do you think it makes sense to move the class mention into its own feature point? What are your thoughts?

```suggestion
1. **Glimmer components** offer a greatly simplified API, for both template and backing class. Say hi to outer HTML semantics and namespaced arguments!
```

Author: Ah, I see. I've always equated Glimmer components with native classes. Yep, I think it'd be good to highlight native classes separately. Thanks!

2. **Modifiers** let you interact with the DOM—animate this element, scroll to that element, autofocus on the first form input, etc. Modifiers work on *any* element.

3. **Tracked properties** simplify syncing the DOM with JavaScript changes. You no longer need to write and maintain dependency lists.

> 

Reviewer: I think it's nice to have a well-rounded number of bullet points (e.g. 3 as you provided or maybe 5), so feel free to skip or edit this suggestion if you feel like it:

```suggestion
4. **Native JavaScript classes** provide you a modern developer experience according to the latest JavaScript standards while building your app.
```

Author: Haha, yeah. I like odd numbers too. Let me see if I can insert a fifth. (I'm also okay with omitting native classes, since they have been present in other frameworks.)

Author: Yeah, I will merge with just 3 bullet points.

> 3. **Tracked properties** simplify keeping the DOM up-to-date with JavaScript changes. You no longer need to write and maintain dependency lists.

Author: ```suggestion
3. **Tracked properties** simplify syncing the DOM with JavaScript changes. You no longer need to write and maintain dependency lists.
```

---

> ## [How to Migrate to Octane 🐹🏗️](https://guides.emberjs.com/release/upgrading/current-edition/)

Reviewer: 🙇‍♀
